src/docs/learn.md

```diff
@@ -0,0 +1,92 @@
+# Learn
+
+`learn` command allows allows training a network from training data.
+
+As all commands in stockfish `learn` can be invoked either from command line (as `stockfish.exe learn ...`, but this is not recommended because it's not possible to specify UCI options before `learn` executes) or in the interactive prompt.
+
+`learn` takes named parameters in form `learn param_1_name param_1_value param_2_name param_2_value ...`. Unrecognized parameters form a list of paths to training data files.
```

Comment:
in the form

Reply:
I think in this case "form" is correct because it meant as in "Unrecognized parameters are parsed as paths and are added to the list of training data files". Though maybe I should use that phrasing?

Reply:
I was talking about the first 'form' i.e. `takes named parameters in the form`  (but I'm no native speaker). The second form is correct.